src/test/java/jalau/at18/azul/FactoryTest.java

```diff
@@ -0,0 +1,56 @@
+package jalau.at18.azul;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
```

Comment:
This import is not being used, please remove

Reply:
Okay I will remove the import